src/main/java/com/xwaffle/universalmarket/config/MarketConfig.java

```diff
@@ -90,7 +91,7 @@ public void populate() {
         get().getNode(new Object[]{"Market", "market-price"}).setValue(1000).setComment("Instead of taxing, take a flat amount of the user before they can sell.");
         get().getNode(new Object[]{"Market", "use-permissions-to-sell"}).setValue(false).setComment("You can give permissions for the amount of items a user can sell. com.xwaffle.universalmarket.addmax.3 allows 3 items for the user");
 
-
+        get().getNode(new Object[]{"Market", "blacklist"}).setValue(new ArrayList<String>()).setComment("Items that users can not sell");
```

Comment:
This adds following 2 lines into `Market {}` block in `config.conf` when it is created.
```
    ...
    # Items that users can not sell
    blacklist=[]
    ...
}
```